src/main/java/com/axonivy/ivy/supplements/logviewer/MainController.java

```diff
@@ -252,6 +264,15 @@ private void configureSearch() {
 		});
 	}
 
+	private void configureFilter(){
```

Comment:
Hi! Should it not be named as  configurePeriodTimeFilter  or something like that?

Reply:
All the methods in this snipplet are named in this way, so would be fine to me